spec/place_order_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'place_order'
+
+describe Order do
+subject(:order) { described_class.new }
+
+  describe "#porder" do
+        it 'acknowledges order' do
+          expect(order.order(["chicken"],[1])).to eq "order placed"
+        end
+  end
+
+  describe '#total' do
+    it 'should display the total of your order' do
+      order.order(["chicken"],[1])
+      expect(order.total).to eq "Your total is £10.00"
+    end
+
+    it 'should raise an error if the order is not in the menu' do
+      expect{order.order(["cheese"],[1])}.
+      to raise_error "Item not available"
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.